frontend/src/components/FavoriteLocationsComponent.tsx

```diff
@@ -44,7 +44,9 @@ export const FavoriteLocationsComponent: React.FC<FavoriteLocationsComponentProp
                   onLocationSelected={props.onLocationSelected}
                   location={location} />
               ))
-              : <IonLabel>Keine favorisierten Locationn vorhanden</IonLabel>
+              : <IonLabel>
+                <p className={styles.no_favorites_text}>Keine favorisierten Locations vorhanden</p>
```

Comment:
Using English and German. The headline of every Favorite Page is also in English. Use German instead.

Reply:
Fixed language aswell as implemented the other proposals